index.html

```diff
@@ -4,20 +4,19 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0"> 
-  <title>My Portfolio</title>
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Karel van Oordt's Personal Portfolio</title>
   <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">  <link rel="stylesheet" href="style.css">
 
 </head>
 
 <body>
   <section class="main-header">
-    
+
       <nav class="navbar-mobile">
           <a class="mobile-logo">My Logo</a>
           
           <a class="toolbar-button" href=#>
-            <img src="images/header/toolbar/Union-button.png" alt="toolbar button">
+            <img src="images/header/toolbar/Union-button.png" alt="toolbar menu button">
```

Comment:
- [ ] For all your alt text, it is a better practice to not include direct descriptions such as the word button. Usually, screen readers automatically provide that information. Remove the word button or image from all your alt texts.